packages/common-all/src/dnode.ts

```diff
@@ -430,7 +429,18 @@ export class NoteUtils {
       NoteUtils.addSchema({ note, schemaModule, schema });
       const maybeTemplate = schema.data.template;
       if (maybeTemplate) {
-        SchemaUtils.applyTemplate({ template: maybeTemplate, note, engine });
+        // TODO: Support xvault with user prompting for this flow
+        const tempNote = _.find(_.values(engine.notes), {
```

Comment:
This should use `NoteUtils.getNoteByFnameFromEngine`.

Reply:
Sounds good